deploy/kubernetes/charts/serai/values.yaml

```diff
@@ -0,0 +1,92 @@
+replicaCount: 1
+
+image:
+  repository: serai
+  pullPolicy: IfNotPresent
+  tag: "dev"
+  ports:
+    - name: port1
+      containerPort: 30333
+      protocol: TCP
+    - name: port2
+      containerPort: 9615
+      protocol: TCP
+    - name: port3
+      containerPort: 9933
+      protocol: TCP
+    - name: port4
+      containerPort: 9944
```

Comment:
ws, yet its 9945 in [this tutorial](https://docs.substrate.io/tutorials/get-started/simulate-network/). I'm not sure if we're 44 or 45 or something else entirely.

Reply:
I used the same ports from Serai's Dockerfile [Dockerfile](https://github.com/serai-dex/serai/blob/7fa78a9ac293aaa0a15c83fb4e6239ed6bf69d87/deploy/serai/Dockerfile#L41EXPOSE%2030333%209615%209933%209944)

Reply:
:+1: Sorry for the confusion on my end.